process/README.md

```diff
@@ -108,7 +108,7 @@ Due to the increased community demands around KubeCon + CloudNativeCon (KCCN), t
 
 For each KubeCon + CloudNativeCon's (KCCN) standalone event (where KCCN is the focus, not a joint event with OSS Summit or other such conference) — currently Europe and North America — the following freeze is applied:
```

Comment:
I believe that KCCN India will meet this criteria from this year.

Reply:
made it clear the freeze doesn't apply to India